keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt

```diff
@@ -97,5 +102,22 @@ class DeliveryConfigController(
     validator.validate(deliveryConfig)
   }
 
-  private val log by lazy { LoggerFactory.getLogger(javaClass) }
+  @Operation(
+    description = "Imports a delivery config manifest from source control."
+  )
+  @PostMapping(
+    path = ["/import"],
+    consumes = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE],
+    produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
+  )
+  // Note: permissions are checked by the importer service
```

Comment:
This is because there's nothing to check the permissions against until we've retrieved the delivery config from SCM.